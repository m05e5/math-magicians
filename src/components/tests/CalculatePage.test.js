import React from 'react';

import CalculatorPage from '../CalculatePage';

const ReactTestRenderer = require('react-test-renderer');

it('Creates a snapshot of the Calculate page and it should match the previous one', () => {
  const tree = ReactTestRenderer.create(<CalculatorPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
