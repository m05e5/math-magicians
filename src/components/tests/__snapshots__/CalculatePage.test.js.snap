// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a snapshot of the Calculate page and it should match the previous one 1`] = `
<div
  className="calculator"
>
  <h3>
    Wecome to our page!
  </h3>
  <div
    className="calc"
  >
    <div
      className="calc-input"
    >
      <p
        id="calc-p"
      >
        
      </p>
    </div>
    <table
      className="calc-buttons"
    >
      <button
        className="btnn"
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="btnn"
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="btnn"
        name="%"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="btnn"
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="btnn"
        name={7}
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="btnn"
        name={8}
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="btnn"
        name={9}
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="btnn"
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="btnn"
        name={4}
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="btnn"
        name={5}
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="btnn"
        name={6}
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="btnn"
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="btnn"
        name={1}
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="btnn"
        name={2}
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="btnn"
        name={3}
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="btnn"
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="btnn"
        name={0}
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="btnn"
        name="."
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="btnn"
        name="="
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </table>
  </div>
</div>
`;
